### ISSUE TITLE
Results folders are not generated on Windows when running the manual scripts

### ISSUE THREAD

**Issue by haytham-salhi** [gridgain/yardstick#11]
Hi, 

I tried to run the manual scripts (the server and driver) on the *_windows *_and everything went fine. It generates the logs folders and there are no exceptions in the logs. However, at the end of running, the results (output) folders are not generated. Any idea? 

P.S: In *_linux *_they are generated properly!

Thank you in advance.

**Comment by yzhdanov** [comment]
Can you please zip the whole directory with all produced output - logs & results - and share it somehow.

Thanks!

**Comment by haytham-salhi** [comment]
Sure

**Comment by haytham-salhi** [comment]
Sure. I've uploaded a compressed folder containing the logs (server and driver), properties file, and the output of console after running the driver. Note that I run it using manual scripts for windows (.bat).

Please note as well that you need to change the extension of the attached to zip so that you can unzip it. It fails when trying to upload it as a zip. I don't know why even though it tells you that it accepts .zip!! So I changed the extension to PNG :). 

Please let me know if you need any further information. 
Waiting for your investigation. 

![issue_results](https://cloud.githubusercontent.com/assets/11832107/14266480/d0c05d52-fac8-11e5-9fba-ad66b2a472f3.PNG)

**Comment by yzhdanov** [comment]
Thanks! We will try reproducing it and come back to you with our findings.

**Comment by haytham-salhi** [comment]
Hello, 

Any updates?

**Comment by ntikhonov** [comment]
Hi @haytham-salhi !
I've tested yardstick on windows 10 and it works fine, the results folders have been created. Please, pay your attention that results folders will be created in current directory, not root yardstick directory.

Please let me know if it works for you.

**Comment by haytham-salhi** [comment]
Hi @ntikhonov 

I tried it on Windows 10. Still no luck though. Here is the console output when running the drivers: 

`Haytham@DESKTOP-NABF4QC /cygdrive/c/MarsProjects/git/distributedproject/Distribu                                                                                                                                         tedCachingPerformanceAnalysis
$ ./bin/benchmark-manual-drivers-start.bat
<01:40:39><yardstick> Using default properties file: config\benchmark-win.proper                                                                                                                                         ties
<01:40:39><yardstick> Starting driver config '--outputFolder results-20160426-01                                                                                                                                         4039 --localBind 127.0.0.1 --port 1234 -d 30 -t 1 -dn HazelcastDriverBenchmark -                                                                                                                                         sn HazelcastServer -ds haytham-salhi-description_2'
<01:40:39><yardstick> Log file: C:\MarsProjects\git\distributedproject\Distribut                                                                                                                                         edCachingPerformanceAnalysis\bin\..\logs-20160426-014039\logs_drivers\0_driver.l                                                                                                                                         og
<01:40:39><yardstick> Driver is started
<01:40:39><yardstick> Driver is stopped
<01:40:39><yardstick> Starting driver config '--outputFolder results-20160426-01                                                                                                                                         4039 --localBind 127.0.0.1 --port 1234 -d 30 -t 2 -dn HazelcastDriverBenchmark -                                                                                                                                         sn HazelcastServer -ds haytham-salhi-description_4'
<01:40:39><yardstick> Log file: C:\MarsProjects\git\distributedproject\Distribut                                                                                                                                         edCachingPerformanceAnalysis\bin\..\logs-20160426-014039\logs_drivers\1_driver.l                                                                                                                                         og
<01:40:39><yardstick> Driver is started
<01:40:39><yardstick> Driver is stopped
<01:42:31><yardstick> All drivers are stopped
`

The logs folder are generated but the output folders mentioned above are no generated!!

**Comment by agoncharuk** [comment]
@haytham-salhi From the output you provided I see that the driver node is stopped the same second as it was started, which indicates that there is an exception during the driver execution or the driver was not started at all due to invalid config or classpath. Please refer to the full driver logs in the logs folder.

**Comment by haytham-salhi** [comment]
@agoncharuk That is not right. The logs are generated and the outputs there are normal and do not indicate to any exception. Seems the running script does not print out the time correctly. If you notice the last line, it's 01:42:31, compared to the first line which is 01:40:39, it took about ~ 2 minutes to complete!!

**Comment by ntikhonov** [comment]
@haytham-salhi Could you try to use yardstick from PowerShell (without cigwin)?

**Comment by haytham-salhi** [comment]
@ntikhonov I tried to run it from power shell, still no luck though. Attached is the output. The logs are generated but the results are not! The folder after --outputFolder is not generated!
![capture](https://cloud.githubusercontent.com/assets/11832107/14868916/932b3b28-0cd0-11e6-9e70-f66058d8cff7.PNG)

**Comment by ntikhonov** [comment]
@haytham-salhi Did you look at `bin` folder? Could you please execute `ls` command after benchmark  stopped?

**Comment by yzhdanov** [comment]
I suppose it is "dir" command for Windows.

**Comment by haytham-salhi** [comment]
@nivanov I'm sure they are not there. I searched whole file system. There is no output foler! 

I'm wondering where is command in the batch that creates the folder after --outputFolder?
